corehq/apps/indicators/utils.py

```diff
@@ -1,10 +1,10 @@
 from couchdbkit import ResourceNotFound
 from django.conf import settings
-from corehq.util.quickcache import quickcache
+from corehq.util.quickcache import skippable_quickcache
 from dimagi.utils.couch.database import get_db
 
 
-@quickcache([], timeout=60)
+@skippable_quickcache([], lambda: settings.UNIT_TESTING, timeout=60)
```

Comment:
i'm not sure i agree that disabling caching is a good thing to do by default for tests. i think it's better that we leave caching enabled and only disable it for functions where we know it doesn't matter (like this one).

for example, when i wrote a lot of the tests for the sql sms backend framework, many of them failed because i was using caching, and that forced me to clear the various caches on save of the different models.  if we had disabled caching by default i wouldn't have even noticed that those errors were happening because the tests wouldn't have failed.

i agree referencing the UNIT_TESTING flag directly is not ideal, but i think we only want to skip caching during tests for certain functions and not all of them, so i can't really think of a great alternative.